### PR TITLE
Feature/fix recipe for mongo 3.2

### PR DESCRIPTION
**Pull request by const-g**
Added support for 3.2

### CODE REVIEW

File: recipes/package_repo.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@
   when 'debian'
     apt_repository 'mongodb' do
       uri node['mongodb3']['package']['repo']['url']
-      distribution "#{node['lsb']['codename']}/mongodb-org/stable"
+      distribution "#{node['lsb']['codename']}/mongodb-org/3.2"
```
Comment (sunggun-yu) [inline recipes/package_repo.rb:34]: @Cog-g 

Thank you for the wonderful PR.
btw, we have conflict PR for this line. #14 
I'm going to merge that and modifying the attribute value with `3.2`
Sounds good? ;-)
And also, I'll (may) try ... 
1. Testing whether this PR support the old mongodb version
2. Adding if condition to set the apt key value for each 3.0.x and 3.2.x (if new key is not working for 3.0.x)

Thank you.

**Comment by jperville** [comment]
Just to bring my grain of salt here,here are the installation instructions for both mongodb 3.0 and 3.2 releases:
- https://docs.mongodb.org/v3.0/tutorial/install-mongodb-on-ubuntu/#import-the-public-key-used-by-the-package-management-system
- https://docs.mongodb.org/v3.2/tutorial/install-mongodb-on-ubuntu/#import-the-public-key-used-by-the-package-management-system

Notice the versioned repository name, path and the different package signing keys.

The known repository paths are:
- "#{node['lsb']['codename']}/mongodb-org/3.0"
- "#{node['lsb']['codename']}/mongodb-org/3.2"
  Respectively. Note that the current value in this cookbook is `"#{node['lsb']['codename']}/mongodb-org/stable"` which seems to be a slightly delayed version of the 3.0 branch; currently I have version 3.0.8 on the /mongodb-org/3.0 repository which is not available yet in the /mongodb-org/stable one.

The signing keys are '7F0CEB10' (for 3.0) and 'EA312927' (for 3.2).

I believe that we could easily switch the repository path and the signing keys in the attributes files on whether the `node['mongodb3']['version']` starts_with?("3.0")`or`== 'latest' or starts_with?("3.2")`

**Comment by jperville** [comment]
As of today (17th December), mongodb seems to have updated the 'stable' repository to point to the 3.2 repository. As a consequence the signing key is currently broken. I extended this PR in my own PR #17 (which also solves the conflict with #14).

**Comment by sunggun-yu** [comment]
Thank you for great PR and Sorry for late merging.
PR has been merged and I'm doing some kitchen testing before releasing new version.

Thank you so much!!

**Pull request merged**